environment.yml

```diff
@@ -1,11 +0,0 @@
-# You can use this file to create a conda environment for this pipeline:
```

Comment:
Same as your environment.yml how come you reomve it completely?

Reply:
ok I see you are trying to use the containers from the base.config

Reply:
See above